applications/plugins/SofaCUDA/CMakeLists.txt

```diff
@@ -167,7 +167,6 @@ set(SOURCE_FILES
     sofa/gpu/cuda/CudaNearestPointROI.cpp
     sofa/gpu/cuda/CudaSphereROI.cpp
 
-    sofa/gpu/cuda/CudaExtraMonitor.cpp
```

Comment:
just a small suggestion but this could be only compiled if the deprecated SofaValidation is found.
This way, the potential dev using this component would be warned and do something about it.
But I am really not sure that there is a lot of people which uses 1) Monitor 2) the CUDA version of it .......

Reply:
Yes I can, what kind of deprecated message you suggest? in the cmakelists, in the .cpp ?